### PR TITLE
add prometheus and grafana

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

File: src/prometheus/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      # - alertmanager:9093
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first_rules.yml"
+  # - "second_rules.yml"
```
Comment (sebright) [inline src/prometheus/prometheus.yml]: Why are lines 12, 16, and 17 commented out?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 - **Synthetic Load Generation:** The application demo comes with a background
   job that creates realistic usage patterns on the website using
   [Locust](https://locust.io/) load generator.
+- **[Prometheus](https://prometheus.io/)/[Grafana](https://grafana.com/) APM:** Frontend(Go)
+  and AdService(Java) are instrumented to export metrics to Prometheus. Grafana service is
+  pre-configured with a Dashboard to show OpenCensus metrics.
```
[the remark this answers is not in the document]
Comment (rghetia) [reply README.md]: yes, Grafana Dashboard show the metrics scraped by Prometheus. I'll clarify the text.

File: src/checkoutservice/main.go [hunk 3]
```diff
@@ -103,7 +147,7 @@ func initTracing() {
 			log.Print("registered stackdriver tracing")
 
 			// Register the views to collect server stats.
```
[the remark this answers is not in the document]
Comment (rghetia) [reply src/checkoutservice/main.go]: yes. I'll remove the comment.

File: src/checkoutservice/Gopkg.lock [hunk 4]
```diff
@@ -2,27 +2,48 @@
...
-  ]
+  packages = ["."]
+  pruneopts = "UT"
   revision = "37aa2801fbf0205003e15636096ebf0373510288"
   version = "v0.5.0"
 
 [[projects]]
+  branch = "master"
+  digest = "1:f5961cd4f136ee9f3a80ca62a05ecc4fb60107d6a323ae258a1c732b76861809"
+  name = "github.com/GoogleCloudPlatform/microservices-demo"
```
[the remark this answers is not in the document]
Comment (rghetia) [reply src/checkoutservice/Gopkg.lock]: This file was auto generated. I'll follow the other commit.

File: src/prometheus/prometheus.yml (hunk 1 above)
Comment (rghetia) [reply src/prometheus/prometheus.yml]: I am not using any alert manager. Same with rules. This was a default template I got from Prometheus.

File: README.md [hunk 5]
```diff
@@ -63,6 +63,10 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 - **Synthetic Load Generation:** The application demo comes with a background
   job that creates realistic usage patterns on the website using
   [Locust](https://locust.io/) load generator.
+- **[Prometheus](https://prometheus.io/)/[Grafana](https://grafana.com/) APM:** Frontend(Go)
+  and AdService(Java) are instrumented to export metrics to Prometheus. Grafana service scraps
```
Comment (sebright) [inline README.md]: s/scraps/scrapes

**Pull request merged**